typescript/libs/global_vars.py

```diff
@@ -2,13 +2,14 @@
 import re
 
 import sublime
+from os.path import dirname
 from .logger import *
```

Comment:
I wonder why it didn't give me an error... But you are right, better change it to ``import logging`` to avoid future problems.